test/animalCount.test.js

```diff
@@ -0,0 +1,35 @@
+const assert = require('assert');
+const zoo = require('../src/zoo');
+
+describe('animalCount', () => {
+  it('test', () => {
+    let actual, expected;
+
+    // sem parâmetros, returna animais e suas quantidades
```

Comment:
```suggestion
    // sem parâmetros, retorna animais e suas quantidades
```

Aha! Caught a sneaky typo that probably hid here since the first cohort 😂